library/ssl_tls.c

```diff
@@ -4205,6 +4208,9 @@ static uint16_t ssl_tls12_preset_default_sig_algs[] = {
 #if defined(MBEDTLS_ECDSA_C)
     MBEDTLS_SSL_TLS12_SIG_AND_HASH_ALG( MBEDTLS_SSL_SIG_ECDSA, MBEDTLS_SSL_HASH_SHA384 ),
 #endif
+#if defined(MBEDTLS_X509_RSASSA_PSS_SUPPORT)
+    MBEDTLS_TLS1_3_SIG_RSA_PSS_RSAE_SHA384,
```

Comment:
Why we use the TLS13 macros in the guard of "#if defined(MBEDTLS_SSL_PROTO_TLS1_2)"?

Reply:
It is a backward compatible improvement. And the name is an IANA value come from RFC8446. 
The signature algorithm will be sent by OpenSSL server. I do not think we should redefine it.

Reply:
It seems still wired, but I am ok with this workaround.